docs/wiki/User Guide/Configurations.md

```diff
@@ -164,6 +164,10 @@ The following configurations are inherited from the open source Kafka client con
 | min.execution.progress.check.interval.ms                          | Double  | N	      | 5000                                                                                                                                                                                                                                                                    	                                                                                                                                           | The minimum execution progress check interval that users can dynamically set the execution progress check interval to.                                                                                                                                                                                                                                                         |
 | slow.task.alerting.backoff.ms                                     | Double  | N	      | 60000                                                                                                                                                                                                                                                                    	                                                                                                                                           | The minimum interval between slow task alerts. This backoff helps bundling slow tasks to report rather than individually reporting them upon detection.                                                                                                                                                                                                                        |
+| concurrency.adjuster.num.min.isr.check                            | Integer | N         | 5                                                                                                                                                                                                                                                                                                                                                                                                                      | The number of times that (At/Under)MinISR status of partitions in the cluster will be checked during each concurrency auto adjustment interval. For example, if the concurrency auto adjustment interval is 6 minutes and this config is 5, then (At/Under)MinISR status of partitions in the cluster will be checked once in every 72 seconds.                                |
```

Comment:
Nit: These settings look relevant to line 164. Maybe we can move them 2 lines up to line 165? Or perhaps we want to always append new features only to the bottom?

Reply:
As long as the config is defined within the right config group (e.g. `Executor Configurations`), I feel the row order within the table should not really matter.